le.sh

```diff
@@ -279,6 +279,9 @@ _initpath() {
   if [ -z "$CA_CERT_PATH" ] ; then
     CA_CERT_PATH="$WORKING_DIR/$domain/ca.cer"
   fi
+  if [ -z "$CERT_CHAIN_PATH" ] ; then
```

Comment:
Well, going this way, why not just concatenate the files externally to this script? For example, my reload command is a script which concatenates both files to make a full cert chain then restarts my servers. I'm not sure this is worth the added complexity when this can be automated already using the reload command argument.